projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -704,7 +730,10 @@ public function render_custom_email_message_form_field( $type ) {
 		 * @return boolean Indicating if the core invitation main should be sent.
 		 */
 		public function should_send_wp_mail_new_user() {
```

Comment:
Let's update the function comment to talk about 'send_user_notification' checkbox